src/movielens_dataset.jl

```diff
@@ -20,11 +20,7 @@ function load_movielens_dataset(zipfile_name, archive_dir, split_ratio=0.10)
     ratings_file_name = joinpath(temp_dir, archive_dir, "ratings.dat")
     if !isfile(movie_file_name)
         println("Extracting movie data...")
-        run(`unzip $download_path $(archive_dir)/movies.dat -d $temp_dir`)
```

Comment:
It looks like you're removing lines 23-27 when I think you should be just replacing line 23 with "InfoZIP.unzip(...". Can you fix?